modules/analytics/src/views/full/index.tsx

```diff
@@ -218,29 +216,41 @@ const Analytics: FC<any> = ({ bp }) => {
   }
 
   const fetchQnaQuestions = async () => {
-    const qnaIds = orderMetrics(getMetric('msg_sent_qna_count'))
-      .filter(m => m.name)
-      .slice(0, 10)
-      .map(m => m.name)
+    const metrics = orderMetrics(getMetric('msg_sent_qna_count').filter(metric => metric.subMetric)).slice(0, 10)
 
-    const fetchedQuestions = await Promise.all(
-      qnaIds.filter(id => !(id in qnaQuestionsCache)).map(id => fetchQnaQuestion(id.replace('__qna__', '')))
-    )
+    const topQnaQuestions = await Promise.all(
```

Comment:
Pretty much every else where we use `Promise.map` instead of promise.all.

Reply:
I don't see a compelling advantage for using `map` over `all`. Also, `map` is not used anywhere in the React app. Using it would bundle Bluebird in the React app, which we may not want.